icode.properties

```diff
@@ -1,11 +1,19 @@
 category=External Analyzers
-description=Import i-Code CNES results for Fortran(77 & 90) and Shell
+description=Analyze your Fortran(77 & 90) and Shell code with i-Code CNES
 homepageUrl=https://github.com/lequal/sonar-icode-cnes-plugin
 archivedVersions=1.0.2,1.1.0
-publicVersions=
+publicVersions=2.0.1
+
+sonar.pluginKey=icode
```

Comment:
This belongs in your project pom, not this file. As extraneous data, it should be removed at your next release, but I don't think it will interfere with this release.

Reply:
Ok we will remove it in the next PR, I open an issue on our project to keep it in mind.

Reply:
Just to make sure: you don't remove it in _your_ project. You _need_ it in your project (presumably). You remove it from _this file._